Pipfile

```diff
@@ -1,27 +1,21 @@
 [[source]]
-
 url = "https://pypi.python.org/simple"
 verify_ssl = true
 name = "pypi"
 
-
 [dev-packages]
-
 pytest = "*"
```

Comment:
Could we use the PR to also push the minimal version the other dev dependencies to something newer?